easybuild/easyconfigs/b/Bio-SamTools/Bio-SamTools-1.43-intel-2017a-Perl-5.24.1.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PerlModule'
+
+name = 'Bio-SamTools'
+version = '1.43'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://search.cpan.org/~lds/Bio-SamTools/'
+description = """This is a Perl interface to the SAMtools sequence alignment interface."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://search.cpan.org/CPAN/authors/id/L/LD/LDS']
+sources = [SOURCE_TAR_GZ]
+
+patches = ['%(name)s-%(version)s.patch']
+
+dependencies = [
+    ('Perl', '5.24.1'),
+    ('BioPerl', '1.7.1', versionsuffix),
+    ('SAMtools', '0.1.17'),
+]
+
+options = {'modulename': 'Bio::DB::Sam'}
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['bam2bedgraph', 'bamToGBrowse.pl', 'chrom_sizes.pl', 'genomeCoverageBed.pl']],
+    'dirs': ['lib/perl5/site_perl/%(perlver)s/x86_64-linux-thread-multi/%s' % x for x in ['auto', 'Bio']],
```

Comment:
you need to use `%%(perlver)s` here since you're also doing string formatting with `% x`...

```
TypeError: format requires a mapping
```